### ISSUE TITLE
Can it support 8GB vram devices?

### ISSUE THREAD

**Issue by johnchen86** [cocktailpeanut/fluxgym#194]
Is it possible to support lower memory devices for LORA training? Such as 8GB VRAM？

**Comment by AllyourBaseBelongToUs** [comment]
when using the lowest setting it does use max 7,4 GB on my RTX 4080, though ive also increased the worker numbers

in advanced settings you are able to enable "low vram" then some will be offloaded to your RAM, if you have enough RAM it then you can use that

**Comment by T4CAi** [comment]
> when using the lowest setting it does use max 7,4 GB on my RTX 4080, though ive also increased the worker numbers
> 
> in advanced settings you are able to enable "low vram" then some will be offloaded to your RAM, if you have enough RAM it then you can use that

Hey, I didn't find any lowVram option, rather low RAM which uses Vram to load the model... I can see in the cmd that "HighVram" in on, how do I change that?

**Comment by jonnytracker** [comment]
yes no low vram option and the training settings show HighVram how to cahnge that
